ReportPortal.Client.Tests/Base/LaunchFixtureBase.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using ReportPortal.Client.Models;
+using ReportPortal.Client.Requests;
+using ReportPortal.Client.Tests;
+
+namespace ReportPortal.Client.Tests
+{
+    public class LaunchFixtureBase: BaseFixture, IDisposable
+    {
+        public Launch Launch { get; set; }
```

Comment:
Let's store here `LaunchId` to see that only this is required for further tests.